app/assets/stylesheets/pages/articles/article-list/_article-list.scss

```diff
@@ -17,66 +17,58 @@
 
 	&__item{
 		display: block;
-		width: 68%;	
-		text-decoration: none;
-		border-bottom: 1px solid $textColorSecondary;
-		padding: 26px 0;
-
-		&:hover .article-list__banner-img{
-			opacity: .8;
-		}
+		width: 68%;
+		margin: 26px 0;
+		border-bottom: 1px solid $postUnderline;
+		padding-bottom: 26px;
 
 		&:last-child{
 			border-bottom: 0;
 			padding-bottom: 0;
 		}
 
 		@media only screen and (max-width: 1170px){
-			width: 86%;
+			width: 90%;
 		}
 
 		@media only screen and (max-width: 920px){
 			width: 100%;
 		}
 	}
 
+	&__content-wrapper{
+		display: flex;
+	}
+
+	&__img-link{
+	&:hover ~.article-list__txt-wrapper .article-list__item-title{
+			color: $colorPrimary;
+		}
+	}
+
 	&__banner-img{
 		background-image: image-url("bg.jpg");
 		background-size: cover;
-		height: 179px;
-		width: 275px;
-		display: inline-block;
-		border-radius: 15px;
+		height: 180px;
+		width: 276px;
+		border-radius: 10px;
 		transition: all .3s ease-in-out;
 
-		@media only screen and (max-width: 920px){
-			width: 35%;
-		}
-
 		@media only screen and (max-width: 640px){
 			height: 143px;
 		}
 
 		@media only screen and (max-width: 480px){
 			display: block;
-			width: 70%;
-			height: 171px;
-		}
-
-		@media only screen and (max-width: 420px){
-			width: 80%;
-		}
-
-		@media only screen and (max-width: 375px){
 			width: 100%;
+			height: 171px;
 		}
 	}
 
 	&__txt-wrapper{
 		width: 62%;
 		display: inline-block;
 		vertical-align: top;
```

Comment:
might you wanna check this? because i think we don't need this property

Reply:
already delete this code